### ISSUE TITLE
Display Errors when combobox search results contain html

### ISSUE THREAD

**Issue by FGrissmer** [Lambda-IT/parlamentsdienste-components#231]
Our client has the requirement of displaying an icon with search results in the combobox

![grafik](https://user-images.githubusercontent.com/126675315/222132102-ab061b20-4e03-4acc-92e7-bc6f35c09d17.png)

This is achieved by putting html code in the label field in the list, in this case a div with a svg an some text.

"<div style=\"display: flex; align-items: center\">" +
                  (hit.recipientType === 'USER' ? this.iconUserRaw : this.iconGroupRaw) +
                  "<span>" + hit.labels[langKey] + "</span>" +
                  "</div>"

This is a suboptimal solution, because whe have to do the "<strong>" marking on our end and if we do backspace we get an empty list. with html in the list

![grafik](https://user-images.githubusercontent.com/126675315/222133096-8cc7904f-bd17-4bd5-afd3-fdc7a4515f01.png)

Also when whe select an entry with a cursor we get html text displayed in the search field

![grafik](https://user-images.githubusercontent.com/126675315/222133400-7100df7c-eb6d-49fa-a5c6-9d919f1822d9.png)

Is it possible to expand the capabilities of the combobox to be either able to handle html in the label field or have an option of displaying an icon?

**Comment by tom-wenger** [comment]
We could do 2 optional properties on the item object who is passed as an array of items to the combobox:
-  iconName: name of the icon of the design system https://parlamentsdienstecore.app.lambda-it.ch/?path=/story/typography-icon--basic
- iconSrc: path to the icon (similar to the Pd-Icon component)
If a iconSrc is provided, the iconName would be ignored.
Is that what you need?

**Comment by tom-wenger** [comment]
Yes exactly. Its going to use pd-icon under the hood anyway so you can use all of those predefined icons. Its better to use them instead of external ressources.
Do you need this feature also in Pd-Dropdown and Pd-Search or just in the Combobox?

**Comment by tom-wenger** [comment]
This feat. is now released. Usage und example unter: https://parlamentsdienstecore.app.lambda-it.ch/?path=/story/forms-inputs-combobox--combobox
